src/utils.ts

```diff
@@ -1,5 +1,5 @@
 export const getChunk = (str: string, size: number): string[] => {
-  return str.match(new RegExp('.{1,' + size + '}', 'g'))
+  return str.replace(' ', '').match(new RegExp('.{1,' + size + '}', 'g'))
```

Comment:
We don't need that we assume that there is no empty spaces in the string:

```suggestion
  return str.match(new RegExp('.{1,' + size + '}', 'g'))
```

Reply:
As it is converted before passing it by props, yes